cfme/ansible_tower/explorer.py

```diff
@@ -126,6 +133,20 @@ def is_displayed(self):
         )
 
 
+class OptionForm(View):
+    dialog_name = TextInput('dialog_name')
+
+
+class TowerExplorerJobServiceDialogView(TowerExplorerView):
+    options = View.nested(OptionForm)
+    save_button = Button('Save')
+
```

Comment:
Q: may be we will need `cancel` button as well? I won't found cancel button form `TowerExplorerView`.